WiserTaskScheduler/AutoUpdater/Services/UpdateService.cs

```diff
@@ -22,14 +22,16 @@ public class UpdateService : IUpdateService
 
     private readonly UpdateSettings updateSettings;
     private readonly ILogger<UpdateService> logger;
+    private readonly ISlackChatService slackChatService;
     private readonly IServiceProvider serviceProvider;
 
     private Version lastDownloadedVersion;
     
-    public UpdateService(IOptions<UpdateSettings> updateSettings, ILogger<UpdateService> logger, IServiceProvider serviceProvider)
+    public UpdateService(IOptions<UpdateSettings> updateSettings, ILogger<UpdateService> logger,ISlackChatService slackChatService,  IServiceProvider serviceProvider)
```

Comment:
I'm not going to reject the PR for this, but but the separation of parameters here is inconsistent, as the `slackChatService` parameter misses a space and `serviceProvider` has two.